extensions/widgets/navbar/navbar.lcb

```diff
@@ -342,6 +374,7 @@ end handler
 
 public handler OnLoad(in pProperties as Array)
 	setNavData(pProperties["items"])
+	put pProperties["theme"] into mWidgetTheme
```

Comment:
as this is a new property, it will break any existing saved navbars because there won't be a "theme" key in the array, so I suggest using:

```
if "theme" is among the keys of pProperties then
    put pProperties["theme"] into mWidgetTheme
end if
```